feature-libs/user/profile/components/register/register.component.ts

```diff
@@ -68,7 +65,7 @@ export class RegisterComponent implements OnInit, OnDestroy {
   );
 
   constructor(
-    protected userRegister: UserRegisterFacade,
+    protected registerComponentService: RegisterComponentService,
     protected globalMessageService: GlobalMessageService,
     protected fb: FormBuilder,
     protected router: RoutingService,
```

Comment:
```suggestion
    protected globalMessageService: GlobalMessageService,
    protected fb: FormBuilder,
    protected router: RoutingService,
    protected registerComponentService: RegisterComponentService,
```

Let's move the new constructor parameter to the last position in constructor parameres
It's because the [schematics for constructor migration](https://github.com/SAP/spartacus/pull/15785/files#diff-376763235e53d104df837029fdaf3789a7a339ed73003a4a5076c1cd944d78abR23-R27) will add the new paramerer as the last one in customer's codebase. 

That's how the `addParams` migration works - it adds a param at the last place.
![image](https://user-images.githubusercontent.com/4001059/185613284-2b0fc807-a744-48b8-b85e-1d710b49749a.png)

Reply:
sure Kris, will update the ordering of `RegisterComponentService`